File: cours-filters-java/src/test/java/fr/isima/controllers/HiControllerTest.java
package fr.isima.controllers;

import org.junit.Before;
import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class HiControllerTest {

    private HiController hiController;

    @Before
    public void setUp() throws Exception {
        this.hiController = new HiController();
    }

    @Test
    public void hi_returnsHi() throws Exception {
        assertThat(this.hiController.hi()).isEqualTo("Hi !");
    }
}

File: cours-filters-kotlin/src/test/kotlin/controllers/HiControllerTest.kt
package controllers

import fr.isima.controllers.HiController
import org.junit.Before
import org.junit.Test

import org.assertj.core.api.Assertions.assertThat

class HiControllerTest {

    private val hiController: HiController = HiController()

    @Test
    fun hi_returnsHi() {
        assertThat(this.hiController.hi()).isEqualTo("Hi !")
    }
}